package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/embed-sdk",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
You also need to run `npm install` to bump the version in package-lock.json. I should probably move to yarn but haven't gotten to that yet.